[2024-02-13T01:21:43.173+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task manual__2024-02-13T01:21:40.423410+00:00 [queued]>
[2024-02-13T01:21:43.182+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task manual__2024-02-13T01:21:40.423410+00:00 [queued]>
[2024-02-13T01:21:43.183+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-13T01:21:43.230+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): third_task> on 2024-02-13 01:21:40.423410+00:00
[2024-02-13T01:21:43.238+0000] {standard_task_runner.py:60} INFO - Started process 706 to run task
[2024-02-13T01:21:43.245+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'manual__2024-02-13T01:21:40.423410+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpde5mlte3']
[2024-02-13T01:21:43.247+0000] {standard_task_runner.py:88} INFO - Job 12: Subtask third_task
[2024-02-13T01:21:43.312+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.third_task manual__2024-02-13T01:21:40.423410+00:00 [running]> on host e97f82a89325
[2024-02-13T01:21:43.402+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shiv' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T01:21:40.423410+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-13T01:21:40.423410+00:00'
[2024-02-13T01:21:43.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-13T01:21:43.405+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Yo-yo! I am the amazing third one here!']
[2024-02-13T01:21:43.422+0000] {subprocess.py:86} INFO - Output:
[2024-02-13T01:21:43.425+0000] {subprocess.py:93} INFO - Yo-yo! I am the amazing third one here!
[2024-02-13T01:21:43.426+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-13T01:21:43.459+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20240213T012140, start_date=20240213T012143, end_date=20240213T012143
[2024-02-13T01:21:43.499+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-13T01:21:43.513+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
